examples/core/list_exports/src/main.rs

```diff
@@ -0,0 +1,87 @@
+use frida::{Frida, Message};
+use lazy_static::lazy_static;
+use std::{thread, time::Duration};
+
+lazy_static! {
+    static ref FRIDA: Frida = unsafe { Frida::obtain() };
+}
+
+fn main() {
+    let pid = 1134009;
```

Comment:
How would this work with a hardcoded pid?

Reply:
I was hoping that whoever runs the test could change the `pid`. I can add a comment if that helps. Does that work?"

Reply:
I will add a code to ask for the pid

Reply:
Ideally, you would determine the pid automatically.

Reply:
I updated the code so people can pass the pid by parameter like in the `console_log` example. I tried attaching to itself but that didn't work.

Reply:
Could pid 0 work here?